tests/aws/test_network_configuration.py

```diff
@@ -166,7 +166,11 @@ def test_off_strategy_with_external_port(
     ):
         external_port = 12345
         monkeypatch.setattr(config, "SQS_ENDPOINT_STRATEGY", "off")
-        monkeypatch.setattr(config, "SQS_PORT_EXTERNAL", external_port)
+        monkeypatch.setattr(
+            config,
+            "LOCALSTACK_HOST",
+            config.HostAndPort(host=localstack_host().host, port=external_port),
+        )
```

Comment:
I think using a custom port as well as custom hostname would make a good addition to the `assert_host_customisation` fixture, though not in this PR.